components/ServiceCard.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <article
+    class="bg-white flex flex-grow p-8 justify-center xl:h-104 lg:p-12"
+    data-aos="fade-up"
+    itemscope
+    itemtype="http://schema.org/Service"
+  >
+    <div class="flex flex-col max-w-2xl w-full">
+      <h3 class="mb-0">
```

Comment:
```suggestion
      <h3 class="mb-0" itemprop="name">
```

I'm not 100% sure, since these are svgs now, but these switch to normal text for screen readers right? If so, I think we want `itemprop="name"` here to indicate this is the service name. We probably should look into this some more on other pages too. I think we have lost some SEO benefits on some of these.

Reply:
Yes it should read off the `title` attribute I put on the SVG's. 

Should `itemprop=name` go in the header or SVG files itself you think? 

And are you getting the SEO stats from Lighthouse or something else? Only seeing this on Lighthouse saying that "learn more" is [too non-descript](https://web.dev/link-text/?utm_source=lighthouse&utm_medium=devtools).
<img width="970" alt="Screen Shot 2022-03-07 at 3 52 49 PM" src="https://user-images.githubusercontent.com/62962173/157138015-93855ecf-4ae9-432b-b8d8-de063f3a01c5.png">

Reply:
I think let's just put it on the `h3` for now, as suggested. I'll go ahead and hit the commit suggestion button. Let's take a look at SEO stuff after we get some of these PRs in 👍